tds_test.go

```diff
@@ -662,6 +662,50 @@ func TestSecureConnection(t *testing.T) {
 	}
 }
 
+func TestTDS8Connection(t *testing.T) {
+	checkConnStr(t)
+	tl := testLogger{t: t}
+	defer tl.StopLogging()
+	SetLogger(&tl)
+
+	dsn := makeConnStr(t)
+	if !strings.HasSuffix(strings.Split(dsn.Host, ":")[0], ".database.windows.net") {
```

Comment:
>ffix(strings.Split(dsn.Host, ":")[0], ".database.windows.net" [](http://example.com/codeflow?start=18&length=61)

can we not test against sql2022 on Windows in appveyor?

Reply:
TrustServerCertificate is not allowed if encrypt=strict. Driver will ignore it or use its default value.